@xen-orchestra/web-core/lib/stores/sidebar.store.ts

```diff
@@ -59,7 +59,16 @@ export const useSidebarStore = defineStore('layout', () => {
       }
 
       isExpanded.value = desktopState && !isMobile
-    }
+    },
+    { immediate: true }
+  )
+
+  watch(
+    () => uiStore.isDesktopLarge,
+    isDesktopLarge => {
+      isExpanded.value = isDesktopLarge
+    },
+    { immediate: true }
```

Comment:
Don't you think you can mutualize this `watch` with the `watch` above?

Also, what is the behavior for `isMobile`?
`const isMobile = computed(() => !isDesktop.value)`
Is that mean, if `isDesktopLarge === true` so `isMobile === true`?